indico/modules/rb/client/js/modules/calendar/CalendarListView.jsx

```diff
@@ -101,26 +110,58 @@ class CalendarListView extends React.Component {
           {moment(day, 'YYYY-MM-DD').format('dddd, LL')}
         </Header>
         <Card.Group itemsPerRow={4} stackable styleName="day-cards-container">
-          {bookings.map(this.renderBooking)}
+          {bookings.map(booking => this.renderBooking(booking, day, linkData))}
         </Card.Group>
         <Divider hidden />
       </div>
     );
   };
 
-  renderBooking = booking => {
+  renderLink = (booking, day, linkData, linkBookingOccurrence) => {
+    if (linkData) {
+      const {reservation} = booking;
+      const boundaries = [moment(linkData.startDt), moment(linkData.endDt)];
+      const datesMatch =
+        moment(booking.startDt).isBetween(...boundaries, undefined, '[]') &&
+        moment(booking.endDt).isBetween(...boundaries, undefined, '[]');
+      const linkBtn = (
+        <Button
+          icon={<Icon name="linkify" />}
+          primary
+          disabled={!!booking.link || !datesMatch}
```

Comment:
What about not rendering the link button at all if it's not possible to link? If you have a lot of bookings, it could be hard to see the difference between active and disabled buttons since the shades of blue are pretty similar.

Reply:
I just removed rendering the button if it cannot be linked.